frb_example/benches/dart/lib/interceptor.io.dart

```diff
@@ -0,0 +1,74 @@
+/// interceptor (on native platforms)
+///
+/// underlying timer is based on [Stopwatch](https://api.dart.dev/dev/2.8.0-dev.7.0/dart-core/Stopwatch-class.html)
+///
+/// mostly a lot of boilerplate so that dart agrees to compile.
+
+import 'dart:ffi' as ffi;
+
+import 'package:uuid/uuid.dart';
+
+import 'ffi.io.dart';
+import 'interceptor.dart';
+export 'interceptor.dart';
+export 'bridge_generated.dart';
+
+class FlutterRustBridgeExampleBenchmarkSuitePlatformBench
+    extends FlutterRustBridgeExampleBenchmarkSuitePlatformBenchBase<FlutterRustBridgeInterceptorStdOutIO,
+        FlutterRustBridgeInterceptorJsonIO> {
+  final FlutterRustBridgeInterceptor _interceptor;
+  FlutterRustBridgeExampleBenchmarkSuitePlatformBench(ffi.DynamicLibrary dylib, bool useJSON)
+      : _interceptor = useJSON
+            ? FlutterRustBridgeInterceptorJsonIO() as FlutterRustBridgeInterceptor<TimeWatch>
```

Comment:
uber-nit: Is it possible we remove the "as"

Reply:
if I do then it doesn't compile.
there's many places around where I had to use `show`/`hide`/`as` to please dart compiler.